src/quantem/tomography/tomography_base.py

```diff
@@ -34,132 +47,132 @@ def __init__(
             A descriptive name for the dataset
         """
 
-        if _token is not self._token:
-            raise RuntimeError(
-                "This class is not meant to be instantiated directly. Use the from_data method."
-            )
+        # if _token is not self._token:
+        #     raise RuntimeError(
+        #         "This class is not meant to be instantiated directly. Use the from_data method."
+        #     )
 
         self._device = device
-        self._tilt_series = tilt_series
-        self._recon_volume = recon_volume
+        self._dataset = dataset
+        self._volume_obj = volume_obj
         self._loss = []
         self._mode = []
 
+        self._hard_constraints = self.DEFAULT_HARD_CONSTRAINTS.copy()
+        self._soft_constraints = self.DEFAULT_SOFT_CONSTRAINTS.copy()
+
+        self._logger = logger
+
     @classmethod
     def from_tilt_series(
```

Comment:
I think "from_tilt_series" is a bit confusing relative to "from_data".  Seeing as how you made a tilt_series class, I would suggest these inits:
.from_tilt_series  - from TiltSeries class
.from_array          - from numpy arrays
.from_file              - from a single file, or perhaps also a list of files (if for example each tilt angle is a separate file)